src/test/java/tests/auth/InstitutionAuthServiceTest.java

```diff
@@ -16,16 +13,10 @@
 public class InstitutionAuthServiceTest {
     Token token = new Token();
     AdminCredentials adminCredentials = new AdminCredentials();
```

Comment:
Metot içerisinde değişen değişkenleri global olarak tanımlamaktansa doğrudan metot içerisinde tanımlamak daha doğru bir yöntemdir, bu sayede hataların önüne geçilmiş olunur

Reply:
@BeforeMethod icinde initialize etmek daha uygun olur sanırım. Bu sekilde
![Screenshot 2024-02-09 at 7 11 28 PM](https://github.com/afet-yonetim-sistemi/ays-be-api-automation/assets/114176660/9b5fbe50-2365-4b20-990e-d57d9a3038d2)